### ISSUE TITLE
'Lite Mode' for slower devices / slower internet speed

### ISSUE THREAD

**Issue by BradB111** [ApolloTVofficial/kamino#130]
**What feature would you like added to the app?**
A list of options for the user to select that could help overcome potential problems as a result of a slower device (maybe those cheap android boxes) or a slower than average internet speed

**Describe the solution you'd like (optional):**
A List of options could include:
- Low resolution Posters (Toggle)
- Low resolution Backdrops/Fanart (Toggle)
- Disable animations (Toggle)
- Link file size cap (User configurable in 'MB)
- Link quality cap (Be able to set max resolution as 480p / 720p / 1080p / 4K)(Allow links that don't specify quality through this filter I think is probably best?)

**Additional context**
I understand that this is not high priority whatsoever but maybe something worth looking at later down the line.

The main idea behind these settings to simply make the base app appeal to a wider user-base rather than someone having to fork the project to fill these potential requirements.

If anyone else can think of any similar setting not listed above let me know and I'll add it above.

**Comment by kevincosmos** [comment]
I think you've combined two issues into one here:
- Users who have slow devices and may want to have a "lite mode" which would disable animations, reduce resolution of images etc to save processing time.
- Users who have slow internet and want to be able to limit the quality of the results.

Your current suggestion does not cater for people who have slow devices but not slow internet, and users who have slow internet but not slow devices.

If this were to be implemented, I'd split this into:
- Settings > Performance/Lite Mode
- Settings > Data Saver

**Comment by BradB111** [comment]
I would agree. Just a brief list that could help both scenarios. Separating in settings would make more sense I suppose.

**Comment by jango0** [comment]
Lite mode is a great idea. I maintain an old device (galaxy nexus with android 7) for my family and apollotv is really slow and laggy. That's expected for a device with only 1gb ram, but things like disable animations and such would be really helpfull.
